vendors/espressif/boards/esp32/aws_demos/config_files/ota_demo_config.h

```diff
@@ -71,6 +71,7 @@
  * bytes are read from network interface. Keeping this timeout to a sufficiently
  * large value so as to account for delay of receipt of a large block of message.
  */
+#undef MQTT_RECV_POLLING_TIMEOUT_MS
```

Comment:
```suggestion
#ifndef MQTT_RECV_POLLING_TIMEOUT_MS
#define MQTT_RECV_POLLING_TIMEOUT_MS ( 1000U )
#endif
```

Consider this suggestion for all occurrences